client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -253,12 +273,6 @@ class EmailProvidersComparison extends React.Component {
 	renderEmailForwardingCard() {
 		const { domain, translate } = this.props;
 
-		const showExpandButton =
-			this.doesDomainHaveEmailForwards( domain ) || this.isDomainEligibleForEmail( domain );
-		const buttonLabel = this.doesDomainHaveEmailForwards( domain )
-			? translate( 'Manage email forwarding' )
-			: translate( 'Add email forwarding' );
-
```

Comment:
I think because of the new management section this code was no longer needed - we would either display the `Email Plan` page if the domain has email forwards, or the `Email Comparison` otherwise.